dist/ExtUtils-ParseXS/lib/perlxs.pod

```diff
@@ -2112,27 +2114,410 @@ than a dMY_CTX in another source file.
+
+=item C<a> and C<A>
+
+These are like C<å> and C<Å> respectively, except the lack of a halo
+indicates that the substitued version isn't fully thread-safe.  If an
+C<L</M>> flag also appears, wrap your calls to this function with the
+Perl-furnished locking macros to get full thread-safety.  Otherwise,
+you'll have to examine the man pages.
+
+=item C<B>
+
+This function returns data in a static buffer which may be destroyed by
+the next call in the same thread, or on some platforms, it may be
+destroyed at any time by another thread executing this function or a
+related one.
+
+Some platforms may allocate the buffer per-thread, so it is thread-safe,
+but you have to use the value (or copy it) before the next call.
+
+There may be a re-entrant function available in which the caller
+furnishes the buffer, and which is thread-safe.  Other flags on the
+entry should indicate this.
+
+=item E
+
+This function accesses the program's environment.
+
+Writing to the environment can create a race with other threads either
+reading or writing the environment at the same time.  If all writers (as
+the core does) protect their accesses by surrounding them with
+
+ ENV_LOCK
+ ...
+ ENV_UNLOCK
+
+then the program is safe with regards to these.  If this is done, then
+read-only accesses can be protected by instead using
+
+ ENV_READ_LOCK
+ ...
+ ENV_READ_UNLOCK
+
+calls surrounding them, which allow for any number of readers at the
+same time.  (A few platforms aren't safe to have multiple concurrent
+readers; for these Perl automatically substitutes an exclusive lock when
+you specify C<ENV_READ_LOCK>, so you don't have to worry about it.)
+
+=item C<L>
+
+This function may read the program's locale.  It should be locale
+thread-safe on systems with thread-safe locales.
+You can wrap it within calls to
+
+ LOCALE_READ_LOCK
+ ...
+ LOCALE_READ_UNLOCK
+
+to make it thread-safe with respect to locale changes by the core in all
+platforms.
+
+If the C<E> flag is also present, you can use
+
+ ENVr_LOCALEr_LOCK
+ ...
+ ENVr_LOCALEr_UNLOCK
+
+or C<ENVw_LOCALEr_LOCK> ...  C<ENVw_LOCALEr_UNLOCK> if the environment
+needs to be locked for exclusive access.
+
+The functions related to formatting likely will not access the locale
+unless there is a C<%> format that is locale-dependent, such as the
+decimal point in floating point conversions, or thousands separators in
+most numeric ones.
+
+=item C<G>
+
+This function accesses global variables, which may be per-thread on
+the platform, but are more likely process-wide.  That means that another
+thread may overwrite those variables before you get a chance to use
+them.  To overcome this, use an exclusive lock, unlocking only after you
+copy to safety the contents of all the variables you need.
+
+Ideally, you would create a mutex for these, but you may be able to use
+the C<ENV_LOCK> for this purpose, provided that using it doesn't unduly
+slow down actual environment accesses.
+
+=item C<Ꞧ>
+
```

Comment:
This is horribly mangled by the github pod viewer, and similarly in the table.

https://github.com/khwilliamson/perl5/blob/perlxs/dist/ExtUtils-ParseXS/lib/perlxs.pod

I don't know if adding an `=encoding` directive would help.  (perlpod says "...if your encoding isn't US-ASCII, then put a C<=encoding I<encodingname>> command very early in the document"

Reply:
Also, I expect it won't display properly by default in a terminal on Windows.